quiz.css

```diff
@@ -0,0 +1,98 @@
+/*quiz.css  1st Quiz - HTML/CSS*/
+
+/* GLOBAL FONT */
+@import url(https://fonts.googleapis.com/css?family=Audiowide|Special+Elite|Cinzel+Decorative:400,700);
```

Comment:
In general, prefer `<link>` over `@import` because link allows for parallel downloads